details_energy.md

```diff
@@ -0,0 +1,177 @@
+
+Concentrated Solar Power (CSP) capacity factors across GCAM regions depend on Direct Normal Irradiance (DNI), which is also derived from the the NASA SSE Release 6.0 Data as described in [Zhang et al. (2010)](details_energy.html#zhang2010). Areas with more than 200 days of low DNI (e.g., cloudy days) are excluded as these are not suitable for large-scale CSP installations. High latitude areas (> 45°) are excluded because DNI data was not available.
+
+#### Geothermal
+
+An additional XML file with Enhanced Geothermal Resources (EGS) is included with the standard GCAM input fileset, but is not included in the default configuration file. EGS expands the geothermal resource base significantly, albeit at higher costs, but is excluded from the default configuration due to the uncertainties about the future availability, effectiveness, and potential costs and risks of the technology.
+
+## Energy Transformation
+
+In energy transformation sectors, the output unit and input unit are EJ (per year), the price unit is 1975$ per GJ of output, and the subsector nest is used for competition between different fuels (or feedstocks). The competition between subsectors takes place according to a calibrated logit sharing function, detailed in [choice function](choice.html). Within the subsectors, there may be multiple competing technologies, where technologies typically represent either different efficiency levels, and/or the application of carbon dioxide capture and storage (CCS). The parameters relevant for technologies in GCAM are identified and explained in [energy technologies](en_technologies.html).
+
+In the schematic of the energy system depicted below, the energy transformation and distribution sectors include all sectors except for the resources (colored red) and the final demands (colored light blue).
+
+<img src="gcam-figs/energy_system_structure.jpg" width="900" height="650" /><br/>
+**Simplified schematic of the energy system in each region, showing the inter-sectoral flows of energy goods in GCAM.**
+{: .fig}
+
+### Electricity
+
+The nesting structure of the electric sector is shown in the figure below, with a focus on one repesentative technology.
+
+<img src="gcam-figs/elec_structure.png" width="900" height="200" /><br/>
+**Schematic showing the nesting structure of the electric sector, with levels for choices between fuels, technologies, and cooling systems. Note that this is a simplification of the actual structure used, which includes "pass-through" sectors, because GCAM sectors are only allowed two levels of nesting.**
```

Comment:
I recommend removing ", because GCAM sectors are only allowed two levels of nesting", because we have subsector nesting now!  The rest is relevant because it does reflect the current electricity sector structure.